projects/plugins/jetpack/modules/likes.php

```diff
@@ -76,20 +84,24 @@ public function __construct() {
 		add_action( 'jetpack_activate_module_likes', array( $this, 'set_social_notifications_like' ) );
 		add_action( 'jetpack_deactivate_module_likes', array( $this, 'delete_social_notifications_like' ) );
 
-		Jetpack::enable_module_configurable( __FILE__ );
+		if ( $this->in_jetpack ) {
+			Jetpack::enable_module_configurable( __FILE__ );
```

Comment:
An alternative would be to add this function to Simple's stub version of the Jetpack class. Or do a somewhat more standard check for Simple (`defined( 'IS_WPCOM' ) && IS_WPCOM`, or use `Automattic\Jetpack\Status\Host` like `( new Host() )->is_wpcom_simple()`).